app/index.js

```diff
@@ -110,6 +110,19 @@ app.model = require('./model');
 app.controller = require('./controller');
 
 /**
+ * Log potentially unhandled promises
+ */
+// TODO: move over to testing suite
+if (app.config.env === 'development') {
+  process.on('unhandledRejection', function(reason, p) {
```

Comment:
hmmm, so this is a catch-all if the controllers don't explicitly `.catch()`? I feel like it'd be better to have some sort of middleware that handles this at the express level instead of hooking into the node event system directly.